threads/src/main/kotlin/pl/edu/pw/ia/threads/application/model/ModeratorsRequests.kt

```diff
@@ -0,0 +1,36 @@
+package pl.edu.pw.ia.threads.application.model
+
+import java.util.UUID
+import pl.edu.pw.ia.shared.domain.command.AddModeratorCommand
+import pl.edu.pw.ia.shared.domain.command.RemoveModeratorCommand
+import pl.edu.pw.ia.shared.security.SecurityContext
+
+data class AddModeratorRequest(
+
+	val threadId: UUID,
+	val subjectAccountId: UUID,
+) {
+	fun toCommand(): AddModeratorCommand =
+		AddModeratorCommand(
+			threadId = threadId,
+			subjectAccountId = subjectAccountId,
+			moderatorId = UUID.randomUUID(),
+			accountId = SecurityContext.getAccountId(),
+		)
+}
+
+data class RemoveModeratorRequest(
+
+	val threadId: UUID,
+	val subjectAccountId: UUID,
```

Comment:
We could leave here only subjectAccountId 
or leave a pair of threadId and accountId, and perform a query to lookup the subjectAccountId

Reply:
Changed to `moderatorId`